lumicks/pylake/force_calibration/calibration_results.py

```diff
@@ -91,12 +91,27 @@ def number_of_samples(self):
         """Number of fitted samples (-)."""
         return self.ps_data.total_sampled_used
 
-    def plot(self):
-        """Plot the fitted spectrum"""
+    def plot(self, show_excluded=False, show_active_peak=False):
+        """Plot the fitted spectrum
+
+        Parameters
+        ----------
+        show_excluded : bool
+            Show fitting regions excluded from the fit
+        show_active_peak : bool
+            Show active calibration peak when available
+        """
         import matplotlib.pyplot as plt
 
-        self.ps_data.plot(label="Data")
+        if show_active_peak and not hasattr(self.model, "output_power"):
+            raise ValueError("This is not an active calibration")
```

Comment:
I was doubting about this too, but the error on the user side in this case is that they specified an incorrect value (they did not attempt to access this attribute themselves). That said, I will add a raises section, since it's generally a good idea to mention those.